requirements/constraints.txt

```diff
@@ -108,7 +108,7 @@ libsass==0.10.0
 click==8.1.6
 
 # pinning this version to avoid updates while the library is being developed
-openedx-learning==0.5.1
+openedx-learning @ git+https://github.com/open-craft/openedx-learning.git@jill/tagging-less-queries
```

Comment:
To do: merge & tag https://github.com/openedx/openedx-learning/pull/157, and update this dependency.